dist/js/autocomplete.js

```diff
@@ -1121,7 +1131,12 @@
 						});
 					}
 
-					if (reg.test(o.label) || reg.test(o.value)) {
+					var searchKeys = JSON.parse(this.config.searchkeys);
```

Comment:
Using the logic in my previous comments, you can now get rid of this JSON parsing and just do:

var searchkeys = this.searchkeys;

Then do your some() function on that. Also, make sure to remove the console.info line.